app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -571,14 +571,8 @@ private fun rateOnGooglePlayHeading(): UiObject {
 }
 
 private fun aboutFirefoxHeading(): UiObject {
-    val aboutFirefoxHeading = mDevice.findObject(UiSelector().text("About $appName"))
-    scrollToElementByText("About $appName")
-    if (!aboutFirefoxHeading.exists()) {
-        settingsList().swipeUp(2)
-        aboutFirefoxHeading.waitForExists(waitingTime)
-    }
-
-    return aboutFirefoxHeading
+    settingsList().scrollToEnd(3)
```

Comment:
If we're applying 3 everywhere I think we should make it a constant and then if we need to tweak it again we just change it one spot. `SETTINGS_MAXSWIPES` or something

Reply:
@AaronMT done. please re-check.